news/_posts/core-weekly/2022-09-05-coreweekly-35-2022.md

```diff
@@ -0,0 +1,194 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 35 of 2022"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2022-09-05
+authors: [ PrestaShop ]
+image: /assets/images/2017/04/core_weekly_banner.jpg
+twitter_image: /assets/images/theme/banner-core-weekly.jpg
+icon: icon-calendar
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 29th of August to Sunday 4th of September 2022.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+## General messages
+
+Dear developers,
+
+PrestaShop 8 beta phase has ended. We plan to release a report summarizing QA's exploratory tests this week and provide information about the next steps. You can follow the [release progress on GitHub](https://github.com/PrestaShop/PrestaShop/issues/26427).
+
+In other news, Maintainer's team released a security patch for the [Product Comments](https://github.com/PrestaShop/productcomments) module. We highly recommend upgrading it.
+
+Last but not least, a reminder about this week's Public Demo that will [take place on Wednesday](https://build.prestashop.com/news/upcoming-demo-6-2022/).
+
+## Releases
+
+* [productcomments](https://github.com/PrestaShop/productcomments): [Release 5.0.2](https://github.com/PrestaShop/productcomments/releases/tag/v5.0.2)
+* [ps_linklist](https://github.com/PrestaShop/ps_linklist): [v5.0.5](https://github.com/PrestaShop/ps_linklist/releases/tag/v5.0.5)
```

Comment:
```suggestion
* [ps_linklist](https://github.com/PrestaShop/ps_linklist): [ Release 5.0.5](https://github.com/PrestaShop/ps_linklist/releases/tag/v5.0.5)
```

Reply:
I think it should be the other way around. We've always had "vX.Y.Z" 🤔

Reply:
Ok, so we need to change productcomments line